packages/base/src/dialogs/symbology/tiff_layer/types/MultibandColor.tsx

```diff
@@ -88,22 +92,26 @@ const MultibandColor = ({
     }
 
     const colorExpr: ExpressionValue[] = ['array'];
-    const rgb: rgbEnum[] = ['red', 'green', 'blue'];
+    const colors: (keyof ISelectedBands)[] = ['red', 'green', 'blue'];
 
-    rgb.forEach(color => {
+    colors.forEach(color => {
       const bandValue = selectedBandsRef.current[color];
       colorExpr.push(bandValue !== 0 ? ['band', bandValue] : 0);
     });
 
-    // Array expression expects 4 values
-    // Last band should be alpha band added by OpenLayers
-    colorExpr.push(['band', numOfBandsRef.current + 1]);
+    // Push alpha if selected, else default to 255
+    if (selectedBandsRef.current.alpha) {
+      colorExpr.push(['band', selectedBandsRef.current.alpha]);
+    } else {
+      colorExpr.push(255); // full opacity fallback
```

Comment:
Should it be `0` instead? That way we're not backward incompatible. The old behavior was to see through the geotiff layer, not fully hiding what's under that layer.